backend/api/projects.py

```diff
@@ -22,16 +22,25 @@ def init_global_manifest_validator():
         schema = json.load(schema_file)
     g.validator = Draft4Validator(schema, format_checker=FormatChecker())
 
+@projects.before_app_first_request
+def init_gloabl_elasticsearch():
+    global es
+    es = Elasticsearch([{'host': 'elasticsearch', 'port': 9200}])
+
+
+@projects.before_app_first_request
+def init_gloabl_mongoclienth():
+    global mongoclient
+    mongoclient = MongoClient('mongodb:27017')
 
 @projects.before_request
 def init_gloabl_elasticsearch():
-    g.es = Elasticsearch([{'host': 'elasticsearch', 'port': 9200}])
+    g.es = es
```

Comment:
I am not sure why you keep the assignments then if they are globally available anyway?